cmd/webpkgserver/webpkgserver.example.toml

```diff
@@ -202,6 +202,89 @@
   # bytes for the OCSP response.
   #AllowTestCert = false
 
+# IMPORTANT NOTE: the support of the ACME protocol and automatic renewal of
+# certificates is currently in the EXPERIMENTAL stage.  Once we have more
+# experience with people using it out in the wild, we will gradually move it to
+# the PRODUCTION stage.
+#
+# This section is meant for ACME config, which allows the Web Packager to use
```

Comment:
We have used "webpkgserver" to reference the application elsewhere in this config file, so I prefer "webpkgserver" over "the Web Packager" or "Web Packager HTTP Server". I'm not saying "webpkgserver" is better or more correct; I just believe it's less confusing to readers if we use the same term consistently.

Reply:
Done.